packages/ata/package.json

```diff
@@ -24,5 +24,8 @@
     "esbuild": "^0.12.9",
     "esbuild-jest": "^0.5.0",
     "jest": "*"
+  },
+  "peerDependencies": {
+    "typescript": "^4.4.4"
```

Comment:
I'm not sure if there's a specific version of typescript to target (or if `"*"` would work), but it's required for the existing tests to pass.